com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -1855,26 +1867,42 @@ DebugUI.Widget makeWidget(string name, VolumeParameter param)
                             }
                         }
                     };
-                    row.opened = true;
+
+                    // Second row, links to volume gameobjects
+                    var row2 = new DebugUI.Table.Row()
+                    {
+                        displayName = "GameObject",
+                        children = { new DebugUI.Value() { getter = () => "" } }
+                    };
 
                     foreach (var volume in volumes)
                     {
                         var profile = volume.HasInstantiatedProfile() ? volume.profile : volume.sharedProfile;
                         row.children.Add(new DebugUI.Value()
                         {
-                            displayName = volume.name + " (" + profile.name + ")",
+                            displayName = profile.name,
                             getter = () =>
                             {
                                 var scope = volume.isGlobal ? "Global" : "Local";
                                 var weight = data.volumeDebugSettings.GetVolumeWeight(volume);
                                 return scope + " (" + (weight * 100f) + "%)";
                             }
                         });
+
+                        row2.children.Add(new DebugUI.ObjectField()
+                        {
+                            displayName = profile.name,
+                            getter = () => volume,
+                            type = typeof(DiffusionProfileSettings)
+                        });
                     }
 
                     row.children.Add(new DebugUI.Value() { displayName = "Default Value", getter = () => "" });
                     table.children.Add(row);
 
+                    row2.children.Add(new DebugUI.Value() { getter = () => "" });
```

Comment:
I would change : "" => string.Empty, is better for the future to know where we are using empty strings